crates/matrix-sdk-base/src/store/memory_store.rs

```diff
@@ -51,11 +51,9 @@ pub struct MemoryStore {
     sync_token: Arc<RwLock<Option<String>>>,
     filters: Arc<DashMap<String, String>>,
     account_data: Arc<DashMap<GlobalAccountDataEventType, Raw<AnyGlobalAccountDataEvent>>>,
-    members: Arc<DashMap<OwnedRoomId, DashSet<OwnedUserId>>>,
     profiles: Arc<DashMap<OwnedRoomId, DashMap<OwnedUserId, MinimalRoomMemberEvent>>>,
     display_names: Arc<DashMap<OwnedRoomId, DashMap<String, BTreeSet<OwnedUserId>>>>,
-    joined_user_ids: Arc<DashMap<OwnedRoomId, DashSet<OwnedUserId>>>,
-    invited_user_ids: Arc<DashMap<OwnedRoomId, DashSet<OwnedUserId>>>,
+    user_ids: Arc<DashMap<OwnedRoomId, DashMap<OwnedUserId, RoomMemberships>>>,
```

Comment:
I think I'd prefer `members` as the name, since this is grouped by rooms and also contains basic membership info.